mylist.rb

```diff
@@ -0,0 +1,28 @@
+class MyList
```

Comment:
- [ ] Kindly import the `MyEnumerable` module using `require_relative`. Now it is raising an error.

![Screenshot from 2021-11-12 21-18-10](https://user-images.githubusercontent.com/64894927/141515566-79c725d9-81cd-459b-bf83-2d1cb51dadfd.png)

Reply:
Fixed 💯